front/src/applications/operationalStudies/components/Scenario/ConflictsList.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import cx from 'classnames';
+import { BsLightningFill } from 'react-icons/bs';
+import { useTranslation } from 'react-i18next';
+
+export type Conflict = {
+  train_ids: string[];
+  conflict_type: string;
+  start_time: string;
+  end_time: string;
+};
+
+type ConflictTableProps = {
+  conflicts: Conflict[];
+  expanded: boolean;
+  toggleConflictsListExpanded: () => void;
+};
+
+function ConflictCard({ conflict }: { conflict: Conflict }) {
+  const { t } = useTranslation(['operationalStudies/scenario']);
+
+  return (
+    <div className="conflict-card">
+      <BsLightningFill color="red" />
+      <div className="conflict-trains">
+        <div className="card-text">{conflict.train_ids[0]}</div>
+        <div className="card-text">{conflict.train_ids[1]}</div>
+      </div>
+      <div className="conflict-type">
+        <p className="card-text">{t(`${conflict.conflict_type}`)}</p>
+      </div>
+      <div className="conflict-times">
+        <div className="card-text start-time">{conflict.start_time}</div>
+        <div className="card-text end-time">{conflict.end_time}</div>
+      </div>
+    </div>
+  );
+}
+
+export default function ConflictsList({
+  conflicts,
+  expanded,
+  toggleConflictsListExpanded,
```

Comment:
```suggestion
  toggleConflictsList,
```
It's only a suggestion :)